Kevin/dev (#4)
InglesTwins/IncludesGraph: README.md

# IncludesGraph

> This is a simple application that takes your C++ project and iterates through all its files and subdirectories, 

Reviewer: ```suggestion
This application that takes a C++ project and iterates through all its files and sub-directories, 
```

reads the header files and translation units, and creats a graph showing the include dependencies of you project.

## Options
- You can choose to exclude std header files

## Usage

> We will assume that the user is on a Linux machine (either WSL, MacOS or Linux).

Reviewer: ```suggestion
It is assumed that the user is on a Linux machine (either WSL, MacOS or Linux).
```

> There is no installing process, just edit you setup file (for Ubuntu this `~/.bashrc`)

Reviewer: ```suggestion
There is no installing process, just edit the setup file (i.e Ubuntu => `~/.bashrc`)
```

> and add the line

Reviewer: ??

```console
ig [path_to_project_dir] [output_file_name](optional)
```
To visualize the output, you should have `dot` install. 

> After `includes_graphs` has completed simply run the command

Reviewer: ```suggestion
To visualize the output, `dot` must be installed. 
Automation will use the return of `build_dependency_tree`(`includes_graphs`) to format and execute the visualization
```
